src/options/App.tsx

```diff
@@ -111,6 +115,23 @@ export const App = () => {
         />
     );
 
+    const helperTextInvalid = (
+        <Split className="pf-u-mt-xs">
+            <SplitItem>
+                <ExclamationCircleIcon
+                    color="var(--pf-global--danger-color--100)"
+                    className="pf-u-mr-xs"
+                />
+            </SplitItem>
+            <SplitItem>
+                <div className="pf-c-form__helper-text pf-m-error">
+                    Provide the URL of your Dev Spaces installation, e.g.,
+                    https://devspaces.mycluster.redhat.com
```

Comment:
I wonder if this should be fully generic, without an actual company here? 
```suggestion
                    https://devspaces.mycluster.mycorp.com
```